editor/editor_node.cpp

```diff
@@ -7061,6 +7061,7 @@ EditorNode::EditorNode() {
 	scene_root_parent->set_v_size_flags(Control::SIZE_EXPAND_FILL);
 
 	scene_root = memnew(SubViewport);
+	scene_root->set_transparent_background(true);
```

Comment:
Why is this line being added?

Reply:
this is for the CanvasItemEditor viewport background color, without it, the background will always be gray and will not follow the theme colors when we change it, we can't override the default_clear_color for the viewport with every release, so it's better to use a transparent background instead for the 2D editor.

Reply:
That is a fine reason. I am curious if it would have any impact on performance or battery life when using the editor on mobile though.

Reply:
It shouldn't have any impact on performance, the rendering server will not draw a clear color for the `CanvasItemEditor` `SubViewport`.